.github/ISSUE_TEMPLATE/request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Request
+about: This is a request for work
+title: '[REQUEST]: '
+labels: 'request'
+assignees: 'leohhhn', 'waymobetta'
+
+---
+
+### Department:
+<!--- Provide the department submitting the requesting -->
+
+### Date of Deliverable (est.):
+<!--- Provide the estimated date of deliverable -->
+
+### Summary
+<!--- Provide a general summary of the issue -->
+
```

Comment:
What if there is a "main" part of the job request? Maybe have "dept, ddl, summary, & ie. request body"?